__tests__/e2e/scripts/setup.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
```

Comment:
For bash scripts it's generally a good practice to add the line:

`set -e`  which will cause the script to exit on error. This [so answer](https://stackoverflow.com/questions/9952177/whats-the-meaning-of-the-parameter-e-for-bash-shell-command-line/9952249#:~:text=2%20Answers&text=The%20%2De%20option%20means%20%22if,t%20a%20real%20%22error%22.) provides a good overview of the implications of this (and a few downsides)